test/relationships/int.spec.ts

```diff
@@ -602,6 +623,48 @@ describe('Relationships', () => {
 
       expect(query.result.docs).toHaveLength(1)
     })
+
+    it('should allow REST querying on polymorphic relationships', async () => {
```

Comment:
This is using a `fetch` request directly, but the `client` is also rest in the test above. The only difference is that `value` comes first in the above query and `relationTo` comes first in this one. I actually think it could be one test and you make the request twice but swap the order. It is literally testing the same functionality and we just need to cover the nuance of handling the property order causing the issue.

I would delete this test and just add the second query and assertion above.